api/applications.go

```diff
@@ -119,7 +209,7 @@ func CreateApplication(c *gin.Context) {
 			return
 		}
 	}
-	am := model.ApplicationModel{
+	am := model.Application{
```

Comment:
I would put the Application into Pending State here. As I see it is never on Pending state.